### DISCUSSION TITLE
App remove or temporarily unpublished

### DISCUSSION THREAD

**Discussion by fattain-naime** [d4rken-org/sdmaid-se#608] [Q&A] [answered]
I'm sorry if I missed any updates or notices. 

I'm unable to find sd maid or sd maid se app in the play store? Is it removed or temporarily unpublished?

**Comment by d4rken** [comment, marked as answer]
My developer account has been terminated, all of my apps are currently unavailable on Google Play. I'm waiting on my appeal.

**Reply by fattain-naime** [reply]
I'm sorry to hear that. 
Hopefully google will reactivate your console account.

**Reply by shuvashish76** [reply]
Track #610 for updates
